input/docs/declarative.md

```diff
@@ -0,0 +1,448 @@
+you trust have access to it.
+
+### Generate a connection token
+
+<div class="code-label session-2">West</div>
+
+Requesting tokens requires the following format YAML file:
+
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      labels:
+        skupper.io/type: connection-token-request
+      name: west-secret
+
+Save as `token-request.yaml`.
+
+    $ oc apply -f token-request.yaml
+
+To verify this step and download the token:
+
+    $ oc get secret  --export -o yaml west-secret > ~/west-secret.yaml
+
+
+### Use the token to form a connection
+
+With the token in hand, you are ready to connect.  Pass the token from
+West to East.
+
+<div class="code-label session-1">East</div>
+
+Apply the token:
+
+    $ oc apply -f ~/west-secret.yaml
+
+### Check the connection
+
+If you completed the [Getting Started using the CLI](./index.html) you can use the `skupper status` command again to see if things have changed.
+If the connection is made, you should see the following output:
+
+<div class="code-label session-2">West</div>
+
+    $ skupper status
+    Skupper enabled for namespace 'west'. It is connected to 1 other site.
+
+<div class="code-label session-1">East</div>
+
+    $ skupper status
+    Skupper enabled for namespace 'east'. It is connected to 1 other site.
+
+## Step 4: Expose your services
+
+You now have a Skupper network capable of multi-cluster communication,
+but no services are attached to it.  This step uses deployment annotations
+to make a Kubernetes deployment on one namespace
+available on all the connected namespaces.
+
+In the examples below, we use the Hello World application to
+demonstrate service exposure.  The same steps apply for your own
+application.
+
+### Deploy the frontend and backend services
+
+Use `kubectl create deployment` to start the frontend in West.
+
+<div class="code-label session-2">West</div>
+
+    kubectl create deployment hello-world-frontend --image quay.io/skupper/hello-world-frontend
+
+Likewise, use `kubectl create deployment` to start the backend in
+East.
+
+<div class="code-label session-1">East</div>
+
+    kubectl create deployment hello-world-backend --image quay.io/skupper/hello-world-backend
+
+### Expose the backend service
+
+At this point, we have the frontend and backend services running, but
+the frontend has no way to contact the backend.  The frontend and
+backend are in different namespaces (and perhaps different clusters),
+and the backend has no public ingress.
+
+Create annotations of the  deployment in East to make
+`hello-world-backend` available in West.
+
+
+<div class="code-label session-1">East</div>
+
+    oc annotate deployment/hello-world-backend skupper.io/proxy="http"
+    oc annotate deployment/hello-world-backend skupper.io/port="8080"
+
+If you check the services in the OpenShift console of the `west` project, you should now see `hello-world-backend`.
+
+### Check the backend service
+
+Use `kubectl get services` in West to make sure the
```

Comment:
Using 'skupper list-exposed' might be a preferred way to check services.

Reply:
I assumed that part of the value of using a kubectl command here was that it shows that the services appear natively (eg would appear in the OpenShift console), not just skupper can 'see'the services

Note to self: `kubectl get services` is in the other Getting Started too...

Reply:
@ajssmith  WDYT? I've not updated based on "it shows k8s rather than skupper visbility and it's in the 'real' getting started"